packages/tools/kadena-cli/src/tx/commands/txCreateTransaction.ts

```diff
@@ -109,6 +115,13 @@ export const createTransactionCommandNew = createCommand(
       );
     }
     const template = await option.template({ stdin });
+    const showHoles = await option.holes();
+
+    if (showHoles.holes === true) {
+      const holes = await getVariablesByTemplate(template.template, { stdin });
```

Comment:
the holes already exist in `template.templateConfig.variables` right?

Reply:
lol missed that, ill update